server/api/routes.js

```diff
@@ -0,0 +1,5 @@
+const { register, login } = require("./controllers/UserControllers");
```

Comment:
This routing module is invoked on the Express app in `server.js` to wire request routes to the appropriate controllers.